docs/guides/server/update-compatibility.adoc

```diff
@@ -16,6 +16,8 @@ Future versions of {project_name} might change that behavior to use additional i
 This is fully scriptable, so your update procedure can use that information to perform a rolling or recreate strategy depending on the change performed.
 It is also GitOps friendly, as it allows storing the metadata of the previous configuration in a file. Use  this file in a CI/CD pipeline with the new configuration to determine if a rolling update is possible or if a recreate update is needed.
 
+NOTE: make sure the build-time state of your `update-compatibility` command matches the build-time state of the corresponding `build` or `start` command you are checking the compatibility of.
+
```

Comment:
We already have this paragraph:

https://github.com/keycloak/keycloak/blob/2091e0541cfc5865a0bdea530fc1e33cc33ab3da/docs/guides/server/update-compatibility.adoc#L67-L72

Adding it here seems to be a bit out-of-context here for me. Is there something missing below that you want to add? 

BTW, "build-time state" might not be something people have a shared understand what it is, and how to ensure it. If the statement needs extension that we use below, can it be shifted in how to ensure the build-state matches?

Reply:
We can just omit the new note. I had added something based upon the reaugmentation possibility, then refined it for the new logic - but should have removed it completely.